.github/workflows/release.yml

```diff
@@ -0,0 +1,114 @@
+---
+name: Release
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+    tags:
+      - '*'
+
+env:
+  GO_VERSION: 1.15
+  GORELEASER_VERSION: v0.138.0
```

Comment:
Actual version is https://github.com/goreleaser/goreleaser/releases/tag/v0.146.0